backend/onedrive/onedrive.go

```diff
@@ -244,6 +300,10 @@ func init() {
 		}, {
 			Name: config.ConfigClientSecret,
 			Help: "Microsoft App Client Secret\nLeave blank normally.",
+		}, {
+			Name:    "national_cloud",
+			Help:    "The national Microsoft cloud server to use?\n 0) Microsoft Cloud Global\n 1) Microsoft Cloud for US Govt (L4)\n 2) Microsoft Cloud for US Govt (L5 DOD)\n 3) Microsoft Cloud for Germany\n 4) Microsoft Cloud for China (Operated by 21Vianet)\n Default 0 - Microsoft Cloud Global",
+			Default: 0,
```

Comment:
I think I'd split this config into two items `auth_url` and `graph_url`

I'd then put your national URLs in as `OptionExamples` here. Here is an example from the s3 backend

https://github.com/rclone/rclone/blob/39319b485881c712b754ba67762398d62c7a1aac/backend/s3/s3.go#L129-L138

This will simplify its meaning for the users and make it so they can configure it themselves at the cost of having two config entries which the user has to choose the same option for each.